### DISCUSSION TITLE
Support for Progressive Web Apps

### DISCUSSION THREAD

**Discussion by VollRagm** [cefsharp/CefSharp#4680] [Q&A]
Does CefSharp offer a way to save and run saved Progressive Web Applications (PWAs)?

**Comment by amaitland** [comment]
There's no PWA specific APIs currently.

From memory a PWA is just a zip with a manifest, so you might be able to download/extract/parse manifest/load content

Having never tried it's hard to give any sort of specific direction.

https://github.com/cefsharp/CefSharp/wiki/General-Usage covers a lot of generic topics
